client/ayon_core/hosts/maya/api/alembic.py

```diff
@@ -0,0 +1,305 @@
+import json
+import logging
+import os
+
+from maya import cmds  # noqa
+
+from ayon_core.hosts.maya.api.lib import evaluation
+
+log = logging.getLogger(__name__)
+
+# The maya alembic export types
+ALEMBIC_ARGS = {
+    "attr": (list, tuple),
+    "attrPrefix": (list, tuple),
+    "autoSubd": bool,
+    "dataFormat": str,
+    "endFrame": float,
+    "eulerFilter": bool,
+    "frameRange": str,  # "start end"; overrides startFrame & endFrame
+    "frameRelativeSample": float,
+    "melPerFrameCallback": str,
+    "melPostJobCallback": str,
+    "noNormals": bool,
+    "preRoll": bool,
+    "preRollStartFrame": int,
+    "pythonPerFrameCallback": str,
+    "pythonPostJobCallback": str,
+    "renderableOnly": bool,
+    "root": (list, tuple),
+    "selection": bool,
+    "startFrame": float,
+    "step": float,
+    "stripNamespaces": bool,
+    "userAttr": (list, tuple),
+    "userAttrPrefix": (list, tuple),
+    "uvWrite": bool,
+    "uvsOnly": bool,
+    "verbose": bool,
+    "wholeFrameGeo": bool,
+    "worldSpace": bool,
+    "writeColorSets": bool,
+    "writeCreases": bool,  # Maya 2015 Ext1+
+    "writeFaceSets": bool,
+    "writeUVSets": bool,   # Maya 2017+
+    "writeVisibility": bool,
+}
+
+
+def extract_alembic(
+    file,
+    attr=None,
+    attrPrefix=None,
+    dataFormat="ogawa",
+    endFrame=None,
+    eulerFilter=True,
+    frameRange="",
+    noNormals=False,
+    preRoll=False,
+    preRollStartFrame=0,
```

Comment:
Pretty sure this functionality does not work?

I'm seeing this in my extraction logs:
```
# Warning: Ignoring unsupported flag: -preRollStartFrame
# Warning: Ignoring unsupported flag: 0
```

Reply:
It seems `preRollStartFrame` is not a job argument but a direct argument to `AbcExport`. See logic [here](https://github.com/tokejepsen/maya-alembic-export/blob/master/alembic_export.py#L544). So the current implementation here is broken.